python/hawkey/query-py.cpp

```diff
@@ -747,7 +748,12 @@ filter_unneeded(PyObject *self, PyObject *args, PyObject *kwds)
     std::unique_ptr<libdnf::Query> self_query_copy(new libdnf::Query(*((_QueryObject *) self)->query));
     gboolean c_debug_solver = debug_solver != NULL && PyObject_IsTrue(debug_solver);
 
-    int ret = hy_filter_unneeded(self_query_copy.get(), *swdb, c_debug_solver);
```

Comment:
Isn't it possible to completely remove hy_filter_unneeded function? It is not used any more and doesn't look like part of API (given that it is declared in hy-query-private.hpp).